openfisca_core/entities/tests/test_role.py

```diff
@@ -0,0 +1,24 @@
+from typing import Any
+
+import pytest
+
+from openfisca_core import entities
+
+from ..typing import HasKey
+
+
+@pytest.fixture
+def entity() -> Any:
```

Comment:
Shouldn't this be
```suggestion
def entity() -> HasKey:
```
or rather, with my suggested rollback:
```suggestion
def entity() -> Entity:
```